pages/common/ssh.md

```diff
@@ -11,13 +11,9 @@
 
 `ssh -i {{path/to/key_file}} {{username}}@{{remote_host}}`
 
-- Connect to a remote server using a specific port:
+- Connect to a remote server using a specific port and run a command:
 
-`ssh {{username}}@{{remote_host}} -p {{2222}}`
-
-- Run a command on a remote server:
-
-`ssh {{remote_host}} {{command -with -flags}}`
+`ssh {{username}}@{{remote_host}} -p {{2222}} {{command -with -flags}}`
```

Comment:
I'm not a fan of combining these examples here. It makes the page more dense and difficult to follow for the beginner.